File: NetworkProviders/Hot-API-Client/Hot.API.Client/Models/SelfTopUp/SelfTopUpRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Hot.API.Client.Models
{
    public class SelfTopUpRequest
    {
        public string TargetMobile { get; set; } = string.Empty;
        public string BillerMobile { get; set; } = string.Empty;
        public decimal Amount { get; set; }
        public string Currency { get; set; } = string.Empty;

    }
}

File: Customer APIs/hot.backend.all/hot.api.backend/Hot.Api.Backend/Models/SelfTopUpRequest.vb
Imports System.ComponentModel.DataAnnotations
Namespace Models
    Public Class SelfTopUpRequest
        <Required>
        <MaxLength(20)>
        Public Property TargetMobile As String
        <Required>
        <MaxLength(20)>
        Public Property BillerMobile As String
        <Required>
        Public Property Amount As Decimal

    End Class

    Public Class SelfTopUpDataRequest
        <Required>
        <MaxLength(20)>
        Public Property TargetMobile As String
        <Required>
        <MaxLength(20)>
        Public Property BillerMobile As String
        <Required>
        Public Property Amount As Decimal
        <Required>
        Public Property BrandId As Integer

    End Class
End Namespace
